### DISCUSSION TITLE
Covariance between eta.Cl and eta.Vc in RxODE?

### DISCUSSION THREAD

**Discussion by newname2023** [nlmixrdevelopment/RxODE#487] [Q&A]
Dear Dr. @mattfidler and nlmixr team,
Thanks for the useful package. I am learning your package for my modeling project and I have a problem. I don't know how to bring covariance or correlation between eta.Cl and eta.V in the code model? You can help me. Many thanks.

**Comment by mattfidler** [comment]
Hi @newname2023 

In general with `RxODE` you would specify the correlation in the `omega` simulation term, it:

```r
omega <- lotri(eta.Cl + etaV ~ sd(cor(3, 0.95, 0.25)))
```

And then simulate/solve

```r
s <- rxSolve(model, params, events, omega=omega)
```

Of course if you use the new `rxode2` you can create a model specification with the inital omega built in by using nlmixr2 type functions:

```r
pheno <- function() {
  ini({
    tcl <- log(0.008) # typical value of clearance
    tv <-  log(0.6)   # typical value of volume
    ## var(eta.cl)
    eta.cl + eta.v ~ c(1, 
                       0.01, 1) ## cov(eta.cl, eta.v), var(eta.v)
                      # interindividual variability on clearance and volume
    add.err <- 0.1    # residual variability
  })
  model({
    cl <- exp(tcl + eta.cl) # individual value of clearance
    v <- exp(tv + eta.v)    # individual value of volume
    ke <- cl / v            # elimination rate constant
    d/dt(A1) = - ke * A1    # model differential equation
    cp = A1 / v             # concentration in plasma
    cp ~ add(add.err)       # define error model
  })
}

f <- rxSolve(pheono, events)
```

**Reply by WXX155367** [reply]
Hi. I would like to ask if I can read covariance matrix from NONMEM? I mean, writing a code to read the data from NONMEM document?
